merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinBindings.java

```diff
@@ -207,7 +207,7 @@ private void getModelEffectiveArguments(final Context ctx) {
       final var arguments = this.missionModelService.getModelEffectiveArguments(missionModelId, input.arguments());
 
       ctx.result(ResponseSerializers.serializeEffectiveArgumentMap(arguments).toString());
-    } catch (final MissingArgumentException ex) {
+    } catch (final MissingArgumentsException ex) {
       ctx.status(200)
```

Comment:
I know this isn't part of your PR, but should this status really be 200 if it's serializing failures? There's more below too.